types/fingerpose/index.d.ts

```diff
@@ -0,0 +1,78 @@
+export enum Finger {
```

Comment:
These shouldn't be enums unless the original code was somehow generated by TypeScript or has the same runtime format.

Upstream, they all consist of objects with values, mappings, functions, etc: https://github.com/andypotato/fingerpose/blob/master/src/FingerDescription.js#L1

They wouldn't be compatible with TypeScript's interpretation of a runtime enum.

Reply:
Fixed. I changed it from enum to objects